web/docs/auth/overview.md

```diff
@@ -313,7 +313,7 @@ export function Main() {
 </Tabs>
 
 :::tip
-Since the `user` prop is only available in a page's React component: use the `user` prop in the page's React component and the `useAuth` hook in any other React component.
+While the `user` prop is only available in an authenticated page's React component, it is the preferred option because we can be certain that the user exists on that page. On the other hand, `useAuth` can be used when the user might not exist, making it better suited for non-authenticated components.
 :::
```

Comment:
I'd move this to the beginning of the chapter. It's best to warn users about a better approach at the beginning.

And we can lose the tip - the text is part of the document's theme.